docs/remote/devcontainer-cli.md

```diff
@@ -126,6 +159,97 @@ For example, you may want to pre-build several images that you then reuse across
 
 That's it!
 
+### Building Multi-Architecture builds
+
+#### Pre-requisites
+1. [Node.js 14+](https://nodejs.org).
+2. [The `docker` CLI](/docs/remote/containers#installation).
+3. [Docker `Buildx`](https://docs.docker.com/buildx/working-with-buildx/)
+4. Docker logged in to a Dockerhub registry.
+
+Ensure that the command `docker buildx` returns the following:
+
+```bash
+Usage:  docker buildx [OPTIONS] COMMAND
+
+Extended build capabilities with BuildKit
+
+Options:
+      --builder string   Override the configured builder instance
+
+Management Commands:
+  imagetools  Commands to work on images in registry
+
+Commands:
+  bake        Build from a file
+  build       Start a build
+  create      Create a new builder instance
+  du          Disk usage
+  inspect     Inspect current builder instance
+  ls          List builder instances
+  prune       Remove build cache
+  rm          Remove a builder instance
+  stop        Stop builder instance
+  use         Set the current builder instance
+  version     Show buildx version information
+
+Run 'docker buildx COMMAND --help' for more information on a command.
+```
+
+#### Process Example
+
+1. Create a basic Dockerfile as below:
+
+    ```Dockerfile
+    #FROM alpine:latest
+    ARG VARIANT=3.13
+    FROM mcr.microsoft.com/vscode/devcontainers/base:0-alpine-${VARIANT}
+    #FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-12
+    CMD echo "Running on $(uname -m)"
+    ```
+
+2. Create a basic `devcontainer.json` as below:
+
+    ```json
+    {
+    "name": "Alpine",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": { "VARIANT": "3.15" },
+        "settings": {},
+        "extensions": [],
+        "remoteUser": "vscode"
+      }
+    }
+    ```
+
+3. Place both files in a `.devcontainer` directory.
+
+4. Run the below command:
+
+    ```bash
+    devcontainer build \
+        --platform linux/amd64,linux/arm64 \
+        --image-name hubusername/multiarch-test:v1 \
+        change-me-to-repository-folder-with-dot-devcontainer
+    ```
+
+#### Breaking down the command
+
+1. `devcontainer build`: As before, this invokes the build command by using Docker under the hood.
+2. `--platform linux/amd64,linux/arm64`: This creates Docker images for the `linux/amd64` and `linux/arm64` architectures using `docker buildx`.
+3. `--image-name hubusername/multiarch-test:v1`: This will tag the image.
```

Comment:
I think we should make sure we highlight that when building multi architecture containers using `--platform`, the `--image-name` option is mandatory because it is used to define the container registry name where the container will be pushed. If `--platform` is not included, the `--image-name` is optional.